ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml

```diff
@@ -153,6 +153,20 @@ tests:
         pod should start\| pod sysctls\| build volumes should mount given secrets
         and configmaps into the build pod
     workflow: openshift-e2e-ibmcloud-heterogeneous
+- as: ocp-e2e-powervs-ovn-multi-ppc64le
+  cron: 20 11,23 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      IBMCLOUD_CIS_CRN: 'crn:v1:bluemix:public:internet-svcs:global:a/3c24cb272ca44aa1ac9f6e9490ac5ecd:9b372a78-cec3-45b7-875e-04ba0270c87d::'
+      JQ_VERSION: "1.6"
+      RESOURCE_GROUP: ipi-cicd-resource-group
+      TERRAFORM_VERSION: 1.5.5
+    workflow: openshift-e2e-powervs-heterogeneous-upi
+  timeout: 6h0m0s
```

Comment:
I'm wondering why this failed... it shouldn't have timedout as quickly as it did.